### PR TITLE
Initial commit of buildable PAC55xx library.

### PR DESCRIPTION
**Pull request by vielster**
Includes make rules, IRQ vector generation and basic linker script.

### CODE REVIEW

**Comment by vielster** [comment]
[PAC55xx Family User Manual](https://active-semi.com/wp-content/uploads/PAC55XX_Family_User_Guide.pdf)

File: lib/pac55xx/pac55xx.ld [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/* Linker script for the Qorvo PAC55xx family of MCUs. (128K flash, 32K RAM). */
```
Comment (karlp) [inline lib/pac55xx/pac55xx.ld]: Just add the parts to the devices.data file instead and drop this file

**Comment by karlp** [comment]
is gpio.c meant to be an empty file?

**Comment by karlp** [comment]
You're also missing it from the documentation tree

File: lib/pac55xx/pac55xx.ld (hunk 1 above)
Comment (vielster) [reply lib/pac55xx/pac55xx.ld]: Done. I couldn't quite gather how to get it to generated the .ld file, but it has been added to devices.data.

**Comment by karlp** [comment]
merged after rebase in dd18b9fd thanks!

**Pull request closed without merging**